src/iop/retouch.c

```diff
@@ -2146,6 +2148,8 @@ static void rt_showmask_callback(GtkToggleButton *togglebutton, dt_iop_module_t
   // if blend module is displaying mask do not display it here
   if(module->request_mask_display && !g->mask_display)
   {
+    dt_control_log(_("display masks cannot be set when mask is being displayed on blend module"));
```

Comment:
maybe change this for consistency reasons with message above to:
"cannot display masks when the blending mask is displayed"

It is also shorter.

Reply:
Done.